src/mame/mame.lst

```diff
@@ -32725,7 +32725,8 @@ redhawk                         // (c) 1997 New Vision Ent. license
 redhawkb                        // (c) 1997
 redhawke                        // (c) 1997 Excellent Co., Ltd license
 redhawkg                        // (c) 1997
-redhawki                        // (c) 1997 Haedong Corp. license
+redhawki                        // (c) 1997 Hae Dong Corp. license
+redhawks                        // (c) 1997 Hae Dong Corp. license
```

Comment:
Do you have a source for this being stylised with the space?  RR rules render names with no space between jaso.

Reply:
Fixed that, and also sorted the new entry alphabetically